components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -301,10 +288,10 @@ public function get_pod_filter( $pod_name = '' ) {
 		if ( isset( $possible_pods[ $pod_name ] ) ) {
 			$this_pod = $possible_pods[ $pod_name ];
 
-			if ( in_the_loop() && ! is_singular() ) {
-				$filter = pods_v( 'archive_filter', $this_pod, $filter, true );
-			} else {
+			if ( is_singular() || is_single() ) {
 				$filter = pods_v( 'single_filter', $this_pod, $filter, true );
+			} elseif ( is_archive() || is_post_type_archive() || is_tax() || in_the_loop() ) {
```

Comment:
This would still change the way it currently works dramatically.
By doing this, all taxonomies will always use the archive filter, even outside the loop.
This is inconsistent with the following overview:
https://github.com/pods-framework/pods/pull/5533#issuecomment-699581377

Reply:
Adding to this.. even inside a loop it could be that a single filter for posts should be used.
A lot of themes (including Genesis) use the loop even for singular pages (just the main content)